src/components/SubTask/SubTaskFilter/SubTaskFilterContainer.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { ButtonGroup } from 'react-bootstrap';
+import { subtaskFilterTypes } from '../subTaskFilterTypes';
+import SubTaskFilterButton from './SubTaskFilterButton';
+
+class SubTaskFilterContainer extends Component {
+  render() {
+    return (
+      <>
```

Comment:
you don't need brackets when fragment is returned, as it' single element

Reply:
Hmm, do u mean "<> </>" brackets? I need them since adjacent elements must be wrapped in an enclosing tag. There are <span> and ButtonGroup elements that I am wrapping.

Reply:
Ok, I got what you mean. Unfortunately, react/jsx-wrap-multilines rule of linter is not accepting this syntax:

render() {
    return  <>
        ....
      </>
  }


Should we disable this rule? Or it will be better to keep following this recommendation?

react/jsx-wrap-multilines:
Wrapping multiline JSX in parentheses can improve readability and/or convenience.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md

Reply:
no problem, you can keep it as it is